@here/harp-mapview/lib/geometry/TileGeometryCreator.ts

```diff
@@ -724,6 +731,32 @@ export class TileGeometryCreator {
                     object.userData.geometryId = srcGeometry.uuid;
                 }
 
+                if (isFillTechnique(technique) && (technique.clipping || technique.transparent)) {
+                    object.onBeforeRender = (_renderer, _scene, _camera, _geometry, _material) => {
+                        if (_material.clippingPlanes === null) {
```

Comment:
i believe we already have some setting `object.onBeforeRender`, this code here may override the callback

Reply:
Good point @robertoraggi, and thanks for the chainCallbacks @zbigg .

@zbigg , what do you mean about this not be required?

Reply:
@nzjony While reading code you've added i spotted that for planar projections, you always set same objects which are constant., so i assume in this case there is no need for `onBeforeRender`.
... at leas this is how i understand code you've written :).

Reply:
@nzjony NVM, sorry i've mistakenly took === as !==.